elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,519 @@
-      color: #{$pfe-color--black};
-      border-bottom: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-var(ui-accent)};
+      border-bottom: pfe-fetch(BorderWidth) pfe-fetch(surface--border-style) pfe-fetch(ui-accent);
     }
   }
 }
-
+ 
 /// ===========================================================================
 /// VERTICAL TAB ORIENTATION
 /// ===========================================================================
 
+:host([pfe-variant="earth"][vertical]) .pfe-tab {
+  display: inline-block !important;
+}
+
 @media screen and (min-width: #{pfe-breakpoint(md)}) {
-  :host([vertical]) {
-    --pfe-tabs__tab--BorderTop: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--BorderRight: 0;
-    --pfe-tabs__tab--BorderBottom: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} transparent;
-    --pfe-tabs__tab--Margin: 0 -1px 0 0;
-    --pfe-tabs__tab--PadingLeft: calc(#{pfe-var(container-padding)} * .75);
-    --pfe-tabs__tab--TextAlign: left;
 
-    @include browser-query(ie11) {
-      border: 1px solid transparent;
-      border-right: 0;
-      margin: 0 -1px 0 0;
-      padding-left: 12px;
-      text-align: left;
+  :host {
+    @at-root #{&}([vertical]) {
+      border-bottom-color: transparent;
+      border-bottom-width: 0;
+      border-left-color: pfe-var(surface--border);
+      border-left-width: pfe-var(ui--border-width);
+
+      --pfe-tabs--Margin: 0 -1px 0 0;
+      padding: pfe-var(container-padding);
+
+      @include browser-query(ie11) {
+        border: pfe-fetch(ui--border-width) pfe-fetch(ui--border-style) transparent;
+        border-bottom: 0;
+        border-right: 0;
+        margin: 0 #{pfe-fetch(ui--border-width) * -1} 0 0;
+        padding-left: pfe-fetch(container-padding);
+      }
     }
-  }
 
-  :host([vertical][aria-selected="true"]) {
-    --pfe-tabs__tab--BorderTopColor: transparent;
-    --pfe-tabs__tab--BorderRight: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    @at-root #{&}([vertical][aria-selected="true"]) {
+      border-left-color: pfe-local(highlight);
+      border-left-width: 3px;
 
-    @include browser-query(ie11) {
-      // color: #{$pfe-color--black};
-      border-top-color: transparent;
-      border-right: #{pfe-var(surface--border-width--heavy)} #{pfe-var(surface--border-style)} #{pfe-var(ui-accent)};
+      @include browser-query(ie11) {
+        color: $pfe-color--black;
+        border-top-color: transparent;
+        border-right: pfe-fetch(ui--border-width--active) pfe-var(ui--border-style) pfe-var(ui-accent);
+        border-bottom: 0;
+      }
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"]) {
+      border-left: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
```

Comment:
Interpolation isn't necessary when assigning it to a property.

Reply:
removed